spec/app/domain/authentication/authn_k8s/copy_text_to_file_in_container_spec.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'Authentication::AuthnK8s::CopyTextToFileInContainer' do
+
+  let(:webservice) { double("Webservice") }
+  let(:pod_namespace) { "PodNamespace" }
+  let(:pod_name) { "PodName" }
+  let(:container) { "Container" }
+  let(:path) { "path/to/file" }
+  let(:content) { "Content" }
+  let(:mode) { "Mode" }
+
+  let(:kubectl_exec) { double("KubectlExec") }
+
+  let(:k8s_object_lookup_instance) { double("K8sObjectLookupInstance") }
+  let(:k8s_object_lookup) { double("K8sObjectLookup") }
+  let(:k8s_object_lookup) do
+    double('k8s_object_lookup').tap do |k8s_object_lookup|
+      allow(k8s_object_lookup).to receive(:new)
+        .with(webservice)
+        .and_return(k8s_object_lookup_instance)
+    end
+  end
+
+  before(:each) do
+    allow(kubectl_exec)
+      .to receive(:call)
+  end
+
+  #  ____  _   _  ____    ____  ____  ___  ____  ___
+  # (_  _)( )_( )( ___)  (_  _)( ___)/ __)(_  _)/ __)
```

Comment:
I don't think this file is large enough to warrant a 4-line banner indicating where the tests begin.